web/menu.jspf

```diff
@@ -71,24 +71,153 @@ org.opensolaris.opengrok.web.Util"
             pMap.put(name, esc);
         }
 %>
+<script type="text/javascript" src="<%= request.getContextPath() %>/js/jquery-3.2.0.min.js"></script>
 <script type="text/javascript">/* <![CDATA[ */
-    document.domReady.push(function() { domReadyMenu(); });
+    document.xrefPath = '<%= request.getContextPath() + Prefix.XREF_P
+    %>';document.domReady.push(function() { domReadyMenu(); });
 /* ]]> */
+var initialWidth = null;
```

Comment:
You can duplicate the qtbl ptbl and ltbl in the markup (and use something different than "id" cause it wont be unique anymore) and then hide the ones you dont want (using display: none) and make visible the ones you want (display: block) and the other way round inside some media query

so its not a real "reordering" it is rather showing/hiding the parts you want/dont want to see